supplementary/alica_ros1/alica_ros_turtlesim/libalica-ros-utils/src/WaitForMsg.cpp

```diff
@@ -0,0 +1,35 @@
+#include "WaitForMsg.h"
+
+namespace ros_utils
+{
+
+WaitForMsg::WaitForMsg(alica::BehaviourContext& context)
+        : alica::BasicBehaviour(context)
+{
+}
+
+void WaitForMsg::initialiseParameters()
+{
+    alica::LockedBlackboardRO bb{*getBlackboard()};
```

Comment:
This is only different from WaitForTrigger due to msg type?  Instead of making a new behavior, lets use topic_tools::ShapeShifter to avoid depending on the msg type